boards/ikea-tradfri/include/periph_conf.h

```diff
@@ -38,6 +38,13 @@ extern "C" {
  * @name    Clock configuration
  * @{
  */
+#ifndef CMU_HFXOINIT
+#define CMU_HFXOINIT        CMU_HFXOINIT_DEFAULT
+#endif
+#ifndef CMU_LFXOINIT
+#define CMU_LFXOINIT        CMU_LFXOINIT_DEFAULT
+#endif
```

Comment:
Is this change part of this PR?

Reply:
Yes and no :-)

I saw in @kaibeckmann his port that he deviated from the default values, so I generalized this to be part of `periph_conf.h`. It is true that the board he used isn't part of RIOT-OS (yet), but I thought it was a good addition to take into account.

Reply:
AFAIK you need to be able to override the defaults to setup the radio driver.

Reply:
Thus, all the introduced changes about the clocking and peripherals in general can be considered improvements? If so (and if it's not a pain to take them off this PR), I'd like to have another PR with them.

Reply:
OK, fair enough. I'll open another PR for these changes, then I'll update this PR after that's fixed :-)

Reply:
I think this PR will get in quite fast since the additions for new families and the relocation of some files doesn't change anything of the current working code. I will spend more time reviewing and testing the changes, which is better if they are separated. Thanks!